addon/components/paper-select-header.js

```diff
@@ -0,0 +1,6 @@
+import Ember from 'ember';
+
+
+export default Ember.Component.extend({
```

Comment:
We're not using any `Ember.` in the codebase.
Please `const { Component } = Ember;`